packages/frontend/app/components/school-vocabularies-expanded.js

```diff
@@ -1,14 +1,13 @@
 import Component from '@glimmer/component';
-import { tracked } from '@glimmer/tracking';
+import { cached } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { service } from '@ember/service';
-import { dropTask, restartableTask } from 'ember-concurrency';
-import { map } from 'rsvp';
+import { dropTask } from 'ember-concurrency';
 import { findById } from 'ilios-common/utils/array-helpers';
+import { TrackedAsyncData } from 'ember-async-data';
 
 export default class SchoolVocabulariesExpandedComponent extends Component {
   @service store;
-  @tracked schoolVocabularies = [];
```

Comment:
something broke in this component refactor. when adding and saving a new vocabulary, i get this error coming out of this component in the browser console `Uncaught (in promise) Error: Assertion Failed: Expected id to be a string or number, received null`

Reply:
✅